source/css/scss-common/base/_forms.scss

```diff
@@ -526,197 +539,134 @@ textarea {
+        width: 2rem;
+        height: 2rem;
         background-color: $white;
         background-image: url($images-base-url + 'checkbox-checked.svg');
         background-size: 100%;
       }
+    }
 
-      &:focus~.custom-control-indicator {
+    &:focus~.custom-control-label {
+      &::before {
         box-shadow: none;
       }
     }
   }
 
-  input {
-    &[type='checkbox'],
-    &[type='radio'] {
-      &:focus~span,
-      &:hover~span {
-        &.custom-control-indicator {
-          &:not(.override-focus) {
-            border: 2px solid $blue-dark;
-          }
-        }
+  &.a-form-checkboxes--discret {
+    @include a-fontSize14;
+    padding-left: $spacer * 2.5;
+
+    .custom-control-label {
+      &::before {
+        // Indicator
+        width: $spacer * 1.5;
+        height: $spacer * 1.5;
+        margin-top: -3px;
+        border: 2px solid $blue-darker;
       }
     }
-  }
 
-  &.custom-controls-stacked {
-    .custom-control {
-      display: block;
-      margin-right: 0;
-    }
-  }
-
-  &.a-search-multiple-actor-select {
-    .inactive {
-      cursor: not-allowed;
-
-      span {
-        color: $grey-medium;
+    .custom-control-input {
+      &:checked~.custom-control-label {
+        &::after {
+          top: 2px;
+          width: 1.4rem;
+          height: 1.4rem;
+        }
       }
     }
   }
 
   &.a-search-multiple-actor-select-subunit {
     margin-left: 30px;
   }
+}
 
-  &.a-form-checkboxes--discret {
-    @include a-fontSize14;
+// FORM RADIOBUTTONS (Bootstrap 4)
+.a-custom-radio {
+  padding-bottom: $spacer / 3;
+  padding-left: $spacer * 3;
+  margin-right: $spacer * 2;
+  margin-bottom: $spacer;
+  cursor: pointer;
 
-    .custom-control {
-      padding-left: $spacer * 1.5;
-    }
+  &:last-child {
+    margin-right: 0;
+  }
+
+  .custom-control-label {
+    cursor: pointer;
 
-    .custom-control-indicator {
-      width: $spacer * 1.5;
-      height: $spacer * 1.5;
-      margin-top: -3px;
-      border: 2px solid $blue-darker;
+    &::before {
+      top: $spacer / 12 * -1;
+      width: $spacer * 2;
+      height: $spacer * 2;
+      background-color: $white;
+      border: $spacer / 6 solid $blue;
     }
   }
-}
 
-// FORM RADIOBUTTONS
-.a-radioButtons {
-  .custom-control-indicator {
-    top: $spacer / 12 * -1;
-    width: $spacer * 2;
-    height: $spacer * 2;
-    background-color: $white;
-    border: $spacer / 6 solid $blue;
+  &:hover {
+    .custom-control-label {
+      &::before {
+        background: $white;
+        border: $spacer / 6 solid $blue-darker;
+      }
+    }
   }
 
-  .custom-control+.custom-control {
-    margin-left: 0;
+  .a-radioButtons-title {
+    @extend .a-fontSizeS;
   }
 
-  .custom-control {
-    display: block;
-    padding-bottom: $spacer / 3;
-    padding-left: $spacer * 3;
-    margin-bottom: $spacer;
-    @include media-breakpoint-up( lg ) {
-      display: inline-block;
-      margin-right: $spacer * 2;
-    }
+  .custom-control-input {
```

Comment:
just wondering if we should change 'custom-control' to 'custom-control-input' in line 696 as well?